authn-authz/binding.md

```diff
@@ -76,45 +76,131 @@ $ tanzu auth --kubeconfig /tmp/pinniped_kubeconfig.yaml binding add --user usern
 
 ```
-tanzu auth binding delete --user $user --role $role --namespace $namespace
+tanzu rbac binding delete --user $user --role $role --namespace $namespace
 
-tanzu auth binding delete --group $group --role $role --namespace $namespace
+tanzu rbac binding delete --group $group --role $role --namespace $namespace
 ```
 
 For example:
 
 ```
-$ tanzu auth binding delete --user username@vmware.com --role app-editor --namespace user-ns
+$ tanzu rbac binding delete --user username@vmware.com --role app-editor --namespace user-ns
 ```
+
+### <a id="error-logs"></a> Error logs
+
+#### Permission Denied:
+
+The current user does not have permissions to create or modify rolebinding objects.
+Please use an administrator account when using the rbac cli.
+```
+Error: rolebindings.rbac.authorization.k8s.io "app-operator" is forbidden: User "<subject>" cannot get resource "rolebindings" in API group "rbac.authorization.k8s.io" in the namespace "namespace"
+Usage:
+tanzu rbac binding add [flags]
+Flags:
+-g, --group string User Group
+-h, --help help for add-binding
+-n, --namespace string Namespace
+-r, --role string Role
+-u, --user string User Name
+
+Global Flags:
+--kubeconfig string kubeconfig file
+```
+
+#### Already Bound Error:
+Adding a subject (user or group) to a role that already has the subject will produce the following error:
+```
+Error: User ‘test-user’ is already bound to 'app-operator' role
+Usage:
+tanzu rbac binding add [flags]
+Flags:
+-g, --group string User Group
+-h, --help help for add-binding
+-n, --namespace string Namespace
+-r, --role string Role
+-u, --user string User Name
+
+Global Flags:
+--kubeconfig string kubeconfig file
+```
+
+#### Could Not Find Error:
+Removing a subject (user or group) from a role that does not exist will produce the following error:
+```
+Error: Did not find User 'test-user' in RoleBinding 'app-operator'
+Usage:
+tanzu rbac binding delete [flags]
+
+Flags:
+-g, --group string User Group
+-h, --help help for add-binding
+-n, --namespace string Namespace
+-r, --role string Role
+-u, --user string User Name
+
+Global Flags:
+--kubeconfig string kubeconfig file
+```
+
+#### Object Has Been Modified Error:
+This error is a race condition caused by running the mutiple rbac cli actions at the same time.
+Rerunning the rbac cli may fix the issue.
+```
+Removed User 'test-user' from RoleBinding 'app-operator'
+Removed User 'test-user' from ClusterRoleBinding 'app-operator-cluster-access'
+Error: Operation cannot be fulfilled on rolebindings.rbac.authorization.k8s.io "app-operator": the object has been modified; please apply your changes to the latest version and try again
+Usage:
+tanzu rbac binding delete [flags]
+
+Flags:
+-g, --group string User Group
+-h, --help help for remove-binding
+-n, --namespace string Namespace
+-r, --role string Role
+-u, --user string User Name
+```
+
+### <a id="troubleshooting"></a> Troubleshooting
+
+1) To Get a list of permissions for a user or a group
```

Comment:
Tiny thing, but could the wording be consistent between this point and the next? So either both `To get ...` or `Get ...`